cms/djangoapps/contentstore/config/waffle.py

```diff
@@ -33,6 +33,7 @@ def waffle_flags():
     flag_undefined_default=False
 )
 
+# TODO: After removing this flag, add a migration to remove waffle flag in a separate deployment.
```

Comment:
NIT: Instead of saying `separate` I would say in a `follow up` deployment.

Reply:
FYI: Updated.